test/shared/utils.py

```diff
@@ -79,11 +80,21 @@ def expected_wheels(package_name, package_version, manylinux_versions=['manylinu
     # {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl
     # {python tag} and {abi tag} are closely related to the python interpreter used to build the wheel
     # so we'll merge them below as python_abi_tag
-    python_abi_tags = ['cp27-cp27m', 'cp35-cp35m', 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38',
-                       'pp27-pypy_73', 'pp36-pypy36_pp73']
+
+    python_abi_tags = ['cp35-cp35m', 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38']
+    extra_python_abi_tags = ['cp27-cp27m', 'pp27-pypy_73', 'pp36-pypy36_pp73']
```

Comment:
Maybe `extra_x86_python_abi_tags`, to clarify when they're extra?

Reply:
Done in: https://github.com/joerick/cibuildwheel/pull/273/commits/36e06189929534a603f9a0794d51275008c2e8e0